crates/nu-engine/src/glob_from.rs

```diff
@@ -62,13 +62,13 @@ pub fn glob_from(
         }
 
         (Some(p), just_pattern)
-    } else if path.to_string_lossy().contains('*') {
+    } else if path.to_string_lossy().contains('*') || path.to_string_lossy().contains('?') {
```

Comment:
With the [`Pattern` API](https://doc.rust-lang.org/std/str/pattern/index.html) you can simplify this a bit with a `&[char]`.

To make this whole block of code a bit more robust we could keep that slice in a `const` like `GLOB_CHARS`, so we have less places that could potentially miss this when changing the code.

```suggestion
    } else if path.to_string_lossy().contains(&['*', '?']) {
```

Reply:
Updated!